src/main.rs

```diff
@@ -89,11 +127,61 @@ impl Opts {
 
         path
     }
+
+    // Reads the schema file and returns its contents as a String.
+    pub fn read_schema_to_string(&self) -> Result<String> {
+        match &self.schema_path {
+            Some(schema_path) => {
+                let mut file = File::open(schema_path)
+                    .map_err(|e| anyhow!("Couldn't open schema file {:?}: {}", schema_path, e))?;
+                let mut contents = String::new();
+                file.read_to_string(&mut contents)
+                    .map_err(|e| anyhow!("Couldn't read schema file {:?}: {}", schema_path, e))?;
+                Ok(contents)
+            }
+            None => Err(anyhow!("Schema file path is not provided")),
+        }
+    }
+
+    pub fn read_query_to_string(&self) -> Result<String> {
+        match &self.query_path {
+            Some(query_path) => {
+                let mut file = File::open(query_path)
+                    .map_err(|e| anyhow!("Couldn't open schema file {:?}: {}", query_path, e))?;
+                let mut contents = String::new();
+                file.read_to_string(&mut contents)
+                    .map_err(|e| anyhow!("Couldn't read schema file {:?}: {}", query_path, e))?;
+                Ok(contents)
+            }
+            None => Err(anyhow!("Schema file path is not provided")),
+        }
+    }
 }
 
 fn main() -> Result<()> {
     let opts: Opts = Opts::parse();
 
+    let schema_string = opts.read_schema_to_string()?;
+    let query_string = opts.read_query_to_string()?;
```

Comment:
What do we do if we don't pass both of these? Do we just use the default limits?